xcresultparser/main.swift

```diff
@@ -21,9 +21,15 @@ struct xcresultparser: ParsableCommand {
     @Option(name: .shortAndLong, help: "The name of the project root. If present paths and urls are relative to the specified directory.")
     var projectRoot: String?
     
+    @Option(name: .shortAndLong, help: "Specify which targets to calculate coverage from")
+    var targets: [String] = []
```

Comment:
We should name this 'coverageTargets' to make it clear to the call site, that this is only used for coverage

Reply:
Noted. I gave it 't' as the short name